distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/mock/csw/FederatedCswMockServer.java

```diff
@@ -0,0 +1,207 @@
+        } else {
+            this.httpRoot = httpRoot;
+        }
+
+        this.sourceId = sourceId;
+        this.port = port;
+    }
+
+    /**
+     * Starts the Restito stub server.
+     */
+    public void start() {
+        try {
+            cswStubServer = new StubServer(port).run();
+
+            setDefaultCapabilityResponse();
+            setDefaultQueryResponse();
+        } catch (IOException | RuntimeException e) {
+            fail("Failed to setup " + getClass().getSimpleName() + ": " + e.getMessage());
+        }
+    }
+
+    /**
+     * Stops the Restito stub server.
+     */
+    public void stop() {
+        cswStubServer.stop();
+    }
+
+    /**
+     * Resets the Restito server by stopping and starting it.
+     */
+    public void reset() {
+        // There is no way in version 0.7 of Restito to clear the calls and reset the
+        // StubServer so we need to stop the current one and create a new instance.
+        // TODO - Replace this code when the new version of Restito is used.
+        stop();
+        start();
+    }
+
+    /**
+     * Returns StubHttp's whenHttp method with the stub server as the parameter. Use is the same.
+     * Sets up an Action when a specific Http call is made.
+     *
+     * @return StubHttp using whenHttp method with the stub server as the parameter.
+     */
+    public StubHttp whenHttp() {
+        return StubHttp.whenHttp(cswStubServer);
+    }
+
+    /**
+     * Returns VerifyHttp's verifyHttp method with the stub server as the parameter. Use is the same.
+     * Provides itest verification when a specific Http call is made.
+     *
+     * @return VerifyHttp using verifyHttp method with the tub server as the parameter.
+     */
+    public VerifyHttp verifyHttp() {
+        return VerifyHttp.verifyHttp(cswStubServer);
+    }
+
+    /**
+     * Get the port being used by the stub server.
+     *
+     * @return the port being used by the stub server.
+     */
+    public int getPort() {
+        return port;
+    }
+
+    /**
+     * Get the root url being used by the stub server.
+     *
+     * @return the root url being used by the stub server.
+     */
+    public String getRoot() {
+        return httpRoot;
+    }
+
+    /**
+     * Returns the restito stub server to allow for any unsupported actions
+     *
+     * @return the stub server managed by this class
+     */
+    public StubServer getServer() {
+        return cswStubServer;
+    }
+
+    /**
+     * Set a specific capability Response. Overwrites default/previous response condition.
+     *
+     * @param resourceName name of capability file
+     * @throws IOException
+     */
+    public void setCapabilityResponse(String resourceName) throws IOException {
+        try (InputStream inputStream = getClass().getResourceAsStream("/" + resourceName)) {
+            String document = substituteTags(IOUtils.toString(inputStream));
+
+            LOGGER.debug("Capability response: \n{}", document);
```

Comment:
Using the string interpolation we get from slf4j is fine. It would not break, just not render a new line (even though I thought Java took care of that for you). Either way it's just a debug statement.